gcp_variant_transforms/transforms/variant_to_bigquery.py

```diff
@@ -113,7 +119,22 @@ def expand(self, pcoll):
             self._allow_incompatible_records,
             self._omit_empty_sample_calls))
     if self._limited_write:
-      bq_rows |= 'LimitWrite' >> limit_write.LimitWrite(_WRITE_SHARDS_LIMIT)
+      bq_row_partitions = bq_rows | beam.Partition(random_partition,
```

Comment:
I believe you can actually use a lambda function here:
`beam.Partition(lambda _, n: random.randint(0, n - 1), NUM_PATITIONS)`